Modules/Package.swift

```diff
@@ -5,16 +5,18 @@ import PackageDescription
 let package = Package(
     name: "Modules",
     platforms: [
-        .iOS(.v16) // Keep in sync with Common.xcconfig
+        // Keep in sync with Common.xcconfig
+        .iOS(.v16),
+        .macOS(.v10_14),
+        .watchOS(.v9)
```

Comment:
Technically unrelated, but decided to explicitly add all the platforms declared in `Common.xcconfig`